Makefile.am

```diff
@@ -225,6 +225,40 @@ EXTRA_DIST += \
   javascript/ola/port_table.js \
   javascript/ola/universe_control.js \
   javascript/ola/universe_item.js \
+  javascript/new-src/README.md \
+  javascript/new-src/bower.json \
+  javascript/new-src/src/js/app.js \
```

Comment:
I'm not really a Makefile expert; @nomis52 is probably a better bet, I just don't think they need to be listed in EXTRA_DIST twice, and given the javascript/new-src symlinked version is mostly just for dev's benefit, we could skip it in EXTRA_DIST I think.